Update methods (#17)
d3b-center/OpenPedCan-methods: content/06.methods.md

### EXPERIMENTAL MODEL AND STUDY PARTICIPANT DETAILS
<!-- TODO: add description of all studies here -->

We utilize the Illumina TruSeq Strand Specific Protocol which pulls out poly-adenylated transcripts and builds those into libraries. Please see our methods below:

cDNA Library Construction

> Total RNA was quantified using the Quant-iT™ RiboGreen® RNA Assay Kit and normalized to 5ng/ul. Following plating, 2 uL of ERCC controls (using a 1:1000 dilution) were spiked into each sample. An aliquot of 325 ng for each sample was transferred into library preparation which uses automated variant of the Illumina TruSeq™ Stranded mRNA Sample Preparation Kit. 

Reviewer: ```suggestion
Total RNA was quantified using the Quant-iT™ RiboGreen® RNA Assay Kit and normalized to 5ng/ul. 
Following plating, 2 uL of ERCC controls (using a 1:1000 dilution) were spiked into each sample. 
An aliquot of 325 ng for each sample was transferred into library preparation. 
```

Reviewer: separate into one sentence per line + one suggestion

> This method preserves strand orientation of the RNA transcript. 
> It uses oligo dT beads to select mRNA from the total RNA sample. 
> It is followed by heat fragmentation and cDNA synthesis from the RNA template. 

Reviewer: ```suggestion
```
don't think this is needed

> The resultant 400bp cDNA then goes through dual-indexed library preparation: ‘A’ base addition, adapter ligation using P7 adapters, and PCR enrichment using P5 adapters. After enrichment the libraries were quantified using Quant-iT PicoGreen (1:200 dilution). 

Reviewer: ```suggestion
The resultant 400bp cDNA went through dual-indexed library preparation: ‘A’ base addition, adapter ligation using P7 adapters, and PCR enrichment using P5 adapters. 
After enrichment, the libraries were quantified using Quant-iT PicoGreen (1:200 dilution). 
```

> After normalizing samples to 5 ng/uL, the set was pooled and quantified using the KAPA Library Quantification Kit for Illumina Sequencing Platforms. 

Reviewer: ```suggestion
Samples were normalized to 5 ng/uL. 
The sample set was pooled and quantified using the KAPA Library Quantification Kit for Illumina Sequencing Platforms. 
```

> The entire process is in 96-well format and all pipetting is done by either Agilent Bravo or Hamilton Starlet.

Reviewer: ```suggestion
```
